src/main/java/com/homework/SpringAppTest.java

```diff
@@ -0,0 +1,30 @@
+package com.homework;
+
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class SpringAppTest {
+    public static void main(String[] args) {
+        String configLocation;
+        ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext(
+                "applicationContext.xml"
+        );
+
+        System.out.println( "------------Prototype Bean--------------------");
+        PrototypeBean prototypeBeanMesage = (PrototypeBean) context.getBean("prototypeBean");
```

Comment:
Use the overload method getBean
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/BeanFactory.html#getBean-java.lang.Class-
![image](https://user-images.githubusercontent.com/20267148/56909109-3ce5eb00-6ab0-11e9-901a-14f5f2c462b8.png)